update.coffee

```diff
@@ -0,0 +1,50 @@
+# Run this to update the list of builtin less functions
+
+path = require 'path'
+fs = require 'fs'
+request = require 'request'
+Promise = require 'bluebird'
```

Comment:
This file is not run in atom's context, and when run from node on the command line, it's not there :( Maybe I'm doing it wrong though.